### PR TITLE
arcgis-js-api example

### PR DESCRIPTION
**Pull request by gavinr**
simple example using babel with arcgis-js-api

Thanks @odoe for https://github.com/odoe/generator-arcgis-js-app, which helped me out pulling this together.

@jpeterson please check this out let me know what you think. I'm thinking maybe we remove the stylus part.

### CODE REVIEW

**Comment by gavinr** [comment]
@jpeterson can you please pull this and make sure the build works on your machine? it's fairly barbones at this point, but IS doing the transpile+build ... at least on my machine!

File: example-arcgis-js-api/bower.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "demo-app",
+  "version": "0.0.1",
```
Comment (tomwayson) [inline example-arcgis-js-api/bower.json]: version is depricated: https://github.com/bower/spec/blob/master/json.md#version

**Comment by jpeterson** [comment]
:+1: works for me

maybe just clean up the gruntfile a bit, remove what looks like some old tasks

File: example-arcgis-js-api/bower.json (hunk 1 above)
Comment (gavinr) [inline example-arcgis-js-api/bower.json]: :heart:

**Pull request merged**